test/ctia/task/migration/store_test.clj

```diff
@@ -250,6 +250,11 @@
 (def fixtures-nb 100)
 (def examples (fixt/bundle fixtures-nb false))
 
+(def sample-relationships-1000
+  "1000 real realtionships extracted from INT with ES doc meta"
+  (with-open [rdr (io/reader "./test/data/indices/sample-relationships-1000.json")]
```

Comment:
Please wrap this with-open in a `delay`, this way we avoid repeatedly performing IO on `(refresh)` which can be slow or torture SSDs :)